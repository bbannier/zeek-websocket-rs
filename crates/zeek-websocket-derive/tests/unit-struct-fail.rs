use zeek_websocket_derive::ZeekType;

#[derive(ZeekType)]
struct A;

fn main() {}
