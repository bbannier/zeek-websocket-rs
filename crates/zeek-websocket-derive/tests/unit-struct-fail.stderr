error: only structs with named fields can derive ZeekType
 --> tests/unit-struct-fail.rs:4:1
  |
4 | struct A;
  | ^^^^^^
